ai_sdlc/services/context7_client.py

```
@@ -156,18 +158,26 @@ async def aclose(self) -> None:
+                httpx.ConnectError,
+                httpx.TimeoutException,
+                Context7TimeoutError,
+            ) as e:
                 last_exception = e
                 if attempt < MAX_RETRIES - 1:
-                    wait_time = RETRY_BACKOFF_FACTOR ** attempt
-                    logger.debug(f"Attempt {attempt + 1} failed, retrying in {wait_time}s: {e}")
+                    wait_time = RETRY_BACKOFF_FACTOR**attempt
+                    logger.debug(
+                        f"Attempt {attempt + 1} failed, retrying in {wait_time}s: {e}"
+                    )
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)